testdata/p4_16_samples/pkginfo_annotation.p4

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2019-present Barefoot Networks, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <core.p4>
+#include <v1model.p4>
+
+struct H { };
+struct M { };
+
+parser ParserI(packet_in pk, out H hdr, inout M meta, inout standard_metadata_t smeta) {
+    state start { transition accept; }
+}
+
+control IngressI(inout H hdr, inout M meta, inout standard_metadata_t smeta) {
+    apply { }
+};
+
+control EgressI(inout H hdr, inout M meta, inout standard_metadata_t smeta) {
+    apply { }
+};
+
+control DeparserI(packet_out pk, in H hdr) {
+    apply { }
+}
+
+control VerifyChecksumI(inout H hdr, inout M meta) {
+    apply { }
+}
+
+control ComputeChecksumI(inout H hdr, inout M meta) {
+    apply { }
+}
+
+
+@pkginfo(
+  name = "some_name",
+  organization = "some_org",
+  version = "some_version",
+  contact = "some_contact",
+  url = "some_url"
+)
+@platform_property(
+  multicast_group_table_size = 10,
+  multicast_group_table_total_replicas = 20,
+  multicast_group_table_max_replicas_per_entry = 30
```

Comment:
It would be really wonderful (but certainly not a blocker) if we could support optional, trailing commas here.

Reply:
Agreed, but I believe not currently how the KV parser works, which I just call without modification. @fruffy might know better though?